packages/addons-website/scss/components/_website-header.scss

```diff
@@ -1,9 +1,14 @@
 @import '../prefix';
 
 .#{$prefix}--header--website {
-  z-index: 9999;
+  z-index: 13000;
   background-color: $ui-05;
   border-bottom: 1px solid $ibm-colors__gray-80;
+  transition: 0.35s cubic-bezier(0.2, 0, 0.38, 0.9) top;
```

Comment:
did these transitions come from Shixie? I think there should be a variable for motion stuff.

Reply:
@alisonjoseph thanks! the motion content page listed this equation but not a token, so i just copied the equation. i'll check for the token before removing [WIP]!